.github/workflows/tox.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        pyver: ['2.7', '3.6', '3.7', '3.8']
+        pyver: ['2.7', '3.6', '3.7', '3.8', '3.9']
```

Comment:
Does Python 3.7 or 3.8 still need to be supported? Fedora 33 is already using Python 3.9 and so is RHEL 9 according to https://developers.redhat.com/articles/2021/11/03/red-hat-enterprise-linux-9-beta-here

Reply:
Maybe one of those versions still needs to be used since it is the platform python version for Ubuntu, but I suspect that at least one version can be dropped when moving to 3.9.

Reply:
And reading further into the diff, it seems that 3.7 is not used.

Reply:
> Maybe one of those versions still needs to be used since it is the platform python version for Ubuntu, but I suspect that at least one version can be dropped when moving to 3.9.

What is the harm of keeping 3.7 or 3.8?

Reply:
> And reading further into the diff, it seems that 3.7 is not used.

Not sure what you mean?
`          37) toxenvs="${toxenvs},coveralls" ;;`

Reply:
> > Maybe one of those versions still needs to be used since it is the platform python version for Ubuntu, but I suspect that at least one version can be dropped when moving to 3.9.
> 
> What is the harm of keeping 3.7 or 3.8?

It uses unnecessary resources and adds more noise.

> > And reading further into the diff, it seems that 3.7 is not used.
> 
> Not sure what you mean? ` 37) toxenvs="${toxenvs},coveralls" ;;`

I meant not used for anything specific other than the unit tests. So if it was needed for ansible-lint, then it would make sense to keep it.

Reply:
fixed in https://github.com/linux-system-roles/network/pull/436/commits/59ed55cceb7b3d136d578506cdec4ad3978987e0